src/modules/typeorm/typeorm.service.ts

```diff
@@ -20,7 +21,7 @@ export class TypeOrmService implements TypeOrmOptionsFactory {
       database: this.config.get<string>('DB_NAME'),
       username: this.config.get<string>('DB_USER'),
       password: this.config.get<string>('DB_PASSWORD'),
-      entities: [Product, Category, User, CartItem, ProductInfo, Photo],
+      entities: [Product, ProductCategory, Category, User, CartItem, Order, OrderItem, Photo],
```

Comment:
you can use the pattern, like under this line, to avoid importing all entities

Reply:
I've found one working pattern, now it works